kqueen_ui/config/utils.py

```diff
@@ -37,11 +36,14 @@ def apply_env_changes(config, prefix='KQUEENUI_'):
         config (obj): Connfiguration object. This configuration will updated.
         prefix (str): Prefix for environment variables. Defaults to `KQUEENUI_`.
 
+    return config_file
     """
 
     for name, value in os.environ.items():
         if name.startswith(prefix):
             config_key_name = name[len(prefix):]
+            if re.search('(?i)true|(?i)false', value):
```

Comment:
If we are converting strings to bool, can't we convert json string to list/dict in python and avoid creating unnecessary class?